romio.spec

```
@@ -18,14 +17,18 @@ URL:        http://www.mpich.org/
 %if ("%{?chroot_name}" == "opensuse-leap-15.1-x86_64")
 %define distro leap15
 %else
+%if ("%{?chroot_name}" == "opensuse-leap-15.2-x86_64")
+%define distro leap15
+%else
```

```suggestion
%if ("%{?chroot_name}" == "opensuse-leap-15.1-x86_64") || ("%{?chroot_name}" == "opensuse-leap-15.2-x86_64")
%define distro leap15
%else
```

Although frankly I don't know why the `()` are being used.  Should likely simply be:

```suggestion
%if "%{?chroot_name}" == "opensuse-leap-15.1-x86_64" || "%{?chroot_name}" == "opensuse-leap-15.2-x86_64"
%define distro leap15
%else
```